.github/autobuild/android.sh

```diff
@@ -0,0 +1,112 @@
+# Definitions have to stay in a specific order due to dependencies.
+QT_BASEDIR="/opt/Qt"
+ANDROID_BASEDIR="/opt/android"
+export ANDROID_SDK_ROOT="${ANDROID_BASEDIR}/android-sdk"
+COMMANDLINETOOLS_DIR="${ANDROID_SDK_ROOT}"/cmdline-tools/latest/
+ANDROID_NDK_ROOT="${ANDROID_BASEDIR}/android-ndk"
+ANDROID_NDK_HOST="linux-x86_64"
+ANDROID_SDKMANAGER="${COMMANDLINETOOLS_DIR}/bin/sdkmanager"
+export JAVA_HOME="/usr/lib/jvm/java-8-openjdk-amd64/"
+export PATH="${PATH}:${ANDROID_SDK_ROOT}/tools"
+export PATH="${PATH}:${ANDROID_SDK_ROOT}/platform-tools"
+
+if [[ ! ${jamulus_buildversionstring:-} =~ [0-9]+\.[0-9]+\.[0-9]+ ]]; then
+    echo "Environment variable jamulus_buildversionstring has to be set to a valid version string"
+    exit 1
+fi
+
+setup_ubuntu_dependencies() {
+    export DEBIAN_FRONTEND="noninteractive"
+
+    sudo apt-get -qq update
+    sudo apt-get -qq --no-install-recommends -y install build-essential zip unzip bzip2 p7zip-full curl chrpath \
+        libxkbcommon-x11-0 openjdk-8-jre-headless openjdk-8-jdk-headless
+}
+
+setup_android_sdk() {
+    mkdir -p "${ANDROID_BASEDIR}"
+
+    if [[ -d "${COMMANDLINETOOLS_DIR}" ]]; then
+        echo "Using commandlinetools installation from previous run (actions/cache)"
+    else
+        mkdir -p "${COMMANDLINETOOLS_DIR}"
+        curl -s -o downloadfile "https://dl.google.com/android/repository/commandlinetools-linux-${COMMANDLINETOOLS_VERSION}_latest.zip"
+        unzip -q downloadfile
+        mv cmdline-tools/* "${COMMANDLINETOOLS_DIR}"
+    fi
+
+    yes | "${ANDROID_SDKMANAGER}" --licenses
+    "${ANDROID_SDKMANAGER}" --update
+    "${ANDROID_SDKMANAGER}" "platforms;${ANDROID_PLATFORM}"
+    "${ANDROID_SDKMANAGER}" "build-tools;${ANDROID_BUILD_TOOLS}"
+}
+
+setup_android_ndk() {
+    mkdir -p "${ANDROID_BASEDIR}"
+    if [[ -d "${ANDROID_NDK_ROOT}" ]]; then
+        echo "Using NDK installation from previous run (actions/cache)"
+    else
+        curl -s -o downloadfile "https://dl.google.com/android/repository/android-ndk-${ANDROID_NDK_VERSION}-linux-x86_64.zip"
+        unzip -q downloadfile
+        mv "android-ndk-${ANDROID_NDK_VERSION}" "${ANDROID_NDK_ROOT}"
+    fi
+}
+
+setup_qt() {
+    if [[ -d "${QT_BASEDIR}" ]]; then
+        echo "Using Qt installation from previous run (actions/cache)"
+    else
+        echo "Installing Qt..."
+        python3 -m pip install "aqtinstall==${AQTINSTALL_VERSION}"
+        python3 -m aqt install-qt --outputdir "${QT_BASEDIR}" linux android "${QT_VERSION}" \
+            --archives qtbase qttools qttranslations qtandroidextras
+    fi
+}
+
+build_app_as_apk() {
+    QT_DIR="${QT_BASEDIR}/${QT_VERSION}/android"
+    MAKE="${ANDROID_NDK_ROOT}/prebuilt/${ANDROID_NDK_HOST}/bin/make"
+
+    "${QT_DIR}/bin/qmake" -spec android-clang
+    "${MAKE}" -j "$(nproc)"
+    "${MAKE}" INSTALL_ROOT=android-build -f Makefile install
+    "${QT_DIR}"/bin/androiddeployqt --input android-Jamulus-deployment-settings.json --output android-build \
```

Comment:
I'd rename android-build to build/ to be consistent to Windows.

Reply:
Done. Let's hope I haven't introduced new errors... Waiting for CI. :)

Reply:
Seems good.